packages/components/pie-textarea/src/index.ts

```diff
@@ -43,21 +67,82 @@ export class PieTextarea extends RtlMixin(LitElement) implements TextareaProps {
         }
     }, 100);
 
+    /**
+     * (Read-only) returns a ValidityState with the validity states that this element is in.
+     * https://developer.mozilla.org/en-US/docs/Web/API/HTMLObjectElement/validity
+     */
+    public get validity (): ValidityState {
+        return (this._textarea as HTMLTextAreaElement).validity;
+    }
+
+    /**
+     * Called after the disabled state of the element changes,
+     * either because the disabled attribute of this element was added or removed;
+     * or because the disabled state changed on a <fieldset> that's an ancestor of this element.
+     * @param disabled - The latest disabled state of the input.
+     */
+    public formDisabledCallback (disabled: boolean): void {
+        this.disabled = disabled;
+    }
+
+    /**
+     * Called when the form that owns this component is reset.
+     * Resets the value to the default value.
+     */
+    public formResetCallback (): void {
+        this.value = defaultProps.value;
```

Comment:
This deviates from the text-input behaviour, which checks if a `defaultValue` prop exits and uses that instead if it does. Which behaviour do we want here?

Reply:
```
export const defaultProps: DefaultProps = {
    value: '',
};
```
by default `value` is an empty string, so when I need to reset form, I equal the current value to an empty string

Reply:
Here is a scenario:

1. I load up a user info page from the server.

2. The `first name` field has a value of `jamie`

3. I edit the field to say `john`

4. I click `reset` on the form

5. The first name input should have a value of `jamie` as this was the initial value. Not an empty string.


Does that make sense? @xander-marjoram just in case you think differently

Reply:
This is a great catch, I think we definitely need to capture this behaviour but I think it's worth its own ticket; I'll write one this morning!

Reply:
Cheers! Happy to approve this PR and address separately in that ticket 👍